otel-collector-config.yaml

```diff
@@ -0,0 +1,146 @@
+# OpenTelemetry Collector Configuration
+# This configuration sets up a basic collector pipeline for traces, metrics, and logs
+
+receivers:
+  # OTLP receiver for gRPC and HTTP protocols
+  otlp:
```

Comment:
@matiasperrone-exo why do u need this file on project root ?
its relatated to container ?
if so should be placed here
https://github.com/OpenStackweb/summit-api/tree/main/docker-compose